api/Window.json

```diff
@@ -5596,6 +5596,44 @@
           }
         }
       },
+      "securitypolicyviolation_event": {
```

Comment:
Per https://w3c.github.io/webappsec-csp/#report-violation it looks like the event can never target `Window`, so please revert this.